src/main/java/org/traccar/protocol/TopinProtocolDecoder.java

```diff
@@ -60,6 +63,21 @@ private void sendResponse(Channel channel, int length, int type, ByteBuf content
         }
     }
 
+    private void updateTime(Channel channel, int msgType){
```

Comment:
Rename `msgType` to `type`. Please use consistent naming.

Reply:
Fixed in commit 6fc7210.